.github/workflows/main.yml

```diff
@@ -0,0 +1,41 @@
+
+name: Kitchen tests
+
+on: [push]
```

Comment:
Will this be triggered on every PR?

Reply:
With the current setting, it will be triggered on any push on every PR.